src/style.css

```diff
@@ -0,0 +1,107 @@
+body {
+  margin: 0;
+  padding: 0;
+  color: #fff;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.columns {
+  display: flex;
+  flex-wrap: wrap;
+}
+  
+.block {
+  display: flex;
+  height: 300px;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 100%;
+  flex-shrink: 0;
+  flex-grow: 1;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);

```

Comment:
It's not a bad solution with :nth-child but let's create a separate class for each block, e.g block-1, block-2 etc. These classes should be responsible for color and order.
And use a mix of classes like `class="block block-1"`